deployment/policies/ecs-service-role-policy.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
A canned version of this policy should already exist within AWS as `arn:aws:iam::aws:policy/service-role/AmazonEC2ContainerServiceRole`. Using that reference is preferable because if they ever extend the functionality, we'll inherit it automatically.